avail/src/main/kotlin/avail/descriptor/fiber/FiberDescriptor.kt

```diff
@@ -371,12 +371,28 @@ class FiberDescriptor private constructor(
 		val debugger = AtomicReference<AvailDebuggerModel?>(null)
 
 		/**
-		 * A function which checks whether the given fiber should run, based on
-		 * what has been set up by the debugger.  This *must* be non-null
-		 * whenever the fiber is captured by a debugger.
+		 * A function which checks whether the fiber in the given interpreter
+		 * should run, based on what has been set up by the debugger.  This
+		 * *must* be non-null whenever the fiber is captured by a debugger.
```

Comment:
Please add some narration to the class comment for `FiberDescriptor` about how debuggability is supported in Avail.